common/config/.golangci.yml

```diff
@@ -242,7 +242,6 @@ linters-settings:
       AllGoFiles:
         files:
           - $all
-          - "!**/tests/integration/**"
```

Comment:
Interestingly, this change does hot show up when running `UPDATE_BRANCH=release-1.19 make update-common` locally. I will create another PR to replace this.